eas-hooks/eas-build-on-success.sh

```diff
@@ -17,20 +33,23 @@ if [[ "$EAS_BUILD_PLATFORM" == "android" ]]; then
     counter=$((counter + 1))
   done
 
+  sleep 10
+
+  # Execute Android tests
   if [[ "$EAS_BUILD_PROFILE" == "test" ]]; then
-    detox test --configuration android.release --headless
+    detox test --configuration android.release --take-screenshots=failing
   fi
   if [[ "$EAS_BUILD_PROFILE" == "test_debug" ]]; then
-    detox test --configuration android.debug --headless
+    detox test --configuration android.debug --take-screenshots=failing
   fi
-
-  # Kill emulator
-  adb emu kill
 else
+  # Execute iOS tests
   if [[  "$EAS_BUILD_PROFILE" == "test" ]]; then
-    detox test --configuration ios.release --headless
+    detox test --configuration ios.release --take-screenshots=failing
   fi
   if [[ "$EAS_BUILD_PROFILE" == "test_debug" ]]; then
-    detox test --configuration ios.debug --headless
+    detox test --configuration ios.debug --take-screenshots=failing
```

Comment:
Is it necessary to pass `--take-screenshots=failing` when it's already configured here - https://github.com/expo/eas-tests-example/pull/8/files#diff-c199d282e0ab6b6dcfee2219852b5d5f2e9a2f3233dcda912eb7bd55d264a42eR15 ?

Reply:
Agreed, the config should automatically be enabling the screenshots